app/models/concerns/has_many_common.rb

```diff
@@ -10,15 +10,20 @@ def name_method
     send(self.class.attribute_name)
```

Comment:
would it make sense to just add `.to_s` here? or is the `attribute_name` called not via `name_method`?